src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -131,7 +131,7 @@ internal abstract class CommonDbContext : DbContext
 
     public DbSet<PackageBlueprintHistory> PackageBlueprintHistory => Set<PackageBlueprintHistory>();
 
-    public DbSet<ComponentSchema> ComponentSchema => Set<ComponentSchema>();
+    public DbSet<ValidatorUptime> ValidatorsUptime => Set<ValidatorUptime>();
```

Comment:
Naming: this table does not store validator uptime; it stores some statistics generated at the end of the epoch; we compute uptime based on those "generic" stats, am I right? 

Sounds like some "validator epoch statistics history" to me?

Reply:
hmm ok, I think we better rename it to ValidatorEmissions, that's how engine calls that.